actions/rules/action_state_to_metastate_test.go

```diff
@@ -62,14 +62,14 @@ func (s *ActionStateToMetastateSuite) TestActionExecution() {
 		// set the state to "in progress" and create a changeset.
 		fxt.WorkItems[0].Fields[workitem.SystemState] = "in progress"
 		contextChanges := []convert.Change{
-			convert.Change{
+			{
 				AttributeName: workitem.SystemState,
-				OldValue: "new",
-				NewValue: "in progress",
+				OldValue:      "new",
```

Comment:
Please use `workitem.SystemStateNew` and so on when dealing with work items created by a fixture. The created work item types all are based on the planner item type and that one is using exactly what we have defined in the `SystemStateNew` constant.

Reply:
I intentionally use literal strings in the tests. Not only in this PR but all I create. Those tests are supposed to fail early. So a typo on the string constant is visible.

Reply:
We don't do that in all the other places. The output of the potentially failing test shows exactly what you need. There's no need to repeat these strings everywhere.

Reply:
Maybe as a compromise you can make it a member function of the work item iteself and add tests to it there. But I also don't like that very much.

Reply:
Outdated. Already fixed.